lib/suspenders/generators/app_generator.rb

```diff
@@ -160,31 +159,29 @@ def install_bitters
     end
 
     def install_refills
-      say "Install Refills"
+      say 'Install Refills'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.